### ISSUE TITLE
Creating Pass on Android 13 with Permission Problem?

### ISSUE THREAD

**Issue by ghost** [ligi/PassAndroid#477]
Hi there!

I'm using
Android 13 with May 2023 Security patch and
PassAndroid v3.7.3 from F-Droid.

I wanted to create a new Pass and encountered the problem, that I couldn't attach a Logo, a footer picture, and an icon. While clicking the button, nothing happens. On older Android versions the file system opens and I can choose a file.

I also cannot import pkpasses with the auto-discover feature for "Looking after passes" in the main Menu by clicking on "+".

Can you fix that please?

**Comment by chipotledonut** [comment]
Same here

I think the app doesn´t ask for file permissions and it can´t be set on settings.

**Comment by easyKL** [comment]
Indeed, the app fails to request permissions. Also storage access cannot be granted.

**Comment by Regenhardt** [comment]
I've been using this app for years, never noticed this problem. I'm not sure if I just never tried though, as I usually just added existing passes into the app. 

Recently upgraded to a new phone with Android 16 though and this works seamlessly. I also have v3.7.3 from F-Droid. Does this problem still persist for you?
